CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## v6.3.1 (Not yet published)
 
+### Fixed
+-   Fixed chevron icon in `<mat-expansion-panel-header>`.
+
+## v6.3.1 (Not yet published)
```

Comment:
Duplicate version here.

Also "fixed chevron" is a little vague; how about:

"Fixed non-center aligned chevron in `<mat-expansion-panel-header>`.

Reply:
Also make sure to merge dev into your current branch; there may be a change conflict now.